### DISCUSSION TITLE
Could we make an exploit with this>

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1380] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So you know how when you go to the website https://robwu.nl/crxviewer/ 

and type in a extension url link to show its files and it has a button that says crx. then when you hit it says app user scripts, and extensions cant be installed from here . does that mean there links that can install chrome extensions not direcly from chrome webstore If so then if we find a link that can do that could It be an exploit ?

![image](https://github.com/user-attachments/assets/5e6adfac-52d2-47d5-93a5-072c3015bd38)

**Comment by S-PScripts** [comment]
i don't think so

**Reply by ghost** [reply]
ok thanks

**Comment by Charlieis1nsan3** [comment]
yeah this kinda just displays the code of an extension.

**Comment by doxjosh** [comment]
You can use this to find the biggest file in your extension and use the print method to freeze the extension

**Reply by ghost** [reply]
ok thanks

**Reply by doxjosh** [reply]
[exploit using this](https://github.com/doxjosh/Print3r)

**Reply by S-PScripts** [reply]
this exploit reminds me of ltprint/anesthesia

**Reply by doxjosh** [reply]
it is that

**Reply by S-PScripts** [reply]
change the name of the exploit to the correct name

**Comment by akabutnicer** [comment]
most of the replies don't mention why this happens. it's a dev tools feature your personal computer has enabled. though on a TYPICAL school computer, the response as in the photo would not be pushed accordingly.
